staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/conversion.go

```diff
@@ -23,25 +23,13 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-// Convertor converts objects to the desired version.
-type Convertor struct {
+// convertor converts objects to the desired version.
+type convertor struct {
 	Scheme *runtime.Scheme
 }
 
-// Convert converts the in object to the out object and returns an error if the
-// conversion fails.
-func (c Convertor) Convert(in runtime.Object, out runtime.Object) error {
-	// For custom resources, because ConvertToGVK reuses the passed in object as
-	// the output. c.Scheme.Convert resets the objects to empty if in == out, so
-	// we skip the conversion if that's the case.
-	if in == out {
```

Comment:
How is this solved today? Does the scheme handle CR conversion now?

Reply:
Added a test case for this behaviour: `scheme.Convert(x, x, nil)` for an unstructured `x` is a no-op.